### DISCUSSION TITLE
PiPedal v1.1.25 arm64

### DISCUSSION THREAD

**Discussion by rerdavies** [rerdavies/pipedal#112] [Releases]
Major features:

- New TooB BF-2 Flanger effect. Emulation of a famous flanger.
- New TooB Stereo Convolution Reverb
- Factory plugin presets for new plugins, and some old plugins.
- Support for state properties in factory presets.

Bug fixes:
- Crash when appending plugins to input nodes, or inserting before end node.
- Write configuration files in indented json format.
- Set default name for plugin preset to name of last-loaded plugin preset.
- Properly release websocket when web page goes into background idle state.

<hr /><em>This discussion was created from the release <a href='https://github.com/rerdavies/pipedal/releases/tag/v1.1.25'>PiPedal v1.1.24 arm64</a>.</em>

**Comment by micahvdm** [comment]
Are you using an RT kernel for PiPedal? If not, check out [Kdoren](https://github.com/kdoren/linux). It's only up to 5.15.65, but its really stable and makes quite the difference to latency.
